aten/src/ATen/native/CPUFallback.cpp

```diff
@@ -183,9 +187,13 @@ void cpu_fallback(const c10::OperatorHandle& op, torch::jit::Stack* stack, bool
   const auto& num_returns = schema_returns.size();
   auto returns = torch::jit::last(stack, num_returns);
   const auto returns_begin = stack->size() - num_returns;
-
-  c10::optional<c10::Device> tgt_device =
-      compute_target_device(tensor_args, tensorlist_args);
+  
+  c10::optional<c10::Device> tgt_device = c10::nullopt;
+  if (compute_target_device(tensor_args, tensorlist_args) != c10::nullopt) {
```

Comment:
nit: assign this to a variable so you don't recompute it?

Reply:
fixed